src/components/Board.js

```diff
@@ -5,29 +5,119 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
   constructor() {
     super();
 
     this.state = {
       cards: [],
+      errorMessages: []
     };
   }
 
+  componentDidMount() {
+    const GET_ALL_CARDS_URL = `https://inspiration-board.herokuapp.com/boards/${this.props.boardName}/cards`;
+
+    axios.get(GET_ALL_CARDS_URL)
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({
+        errorMessages: [...this.state.errorMessages, error.message]
+      });
+    });
+  }
+
+  addCard = (newCard) => {
+    const ADD_CARD = `https://inspiration-board.herokuapp.com/boards/Dionisia/cards?text=${newCard.card.text}&emoji=${newCard.card.emoji}`;
+
+    axios.post(ADD_CARD)
+    .then((response) => {
+      let cardsArray = this.state.cards
+      cardsArray.push(response.data)
+      this.setState({
+        cards: cardsArray
+      });
+    })
+    .catch((error) => {
+      this.setState({
+        errorMessages: error.message
+      })
+    })
+  }
+
+  deleteCard = (id) => {
+    console.log("This card is being deleted!");
+    const DELETE_CARD = `https://inspiration-board.herokuapp.com/cards/${id}`;
+
+    console.log(DELETE_CARD);
+
+    axios.delete(DELETE_CARD)
+    .then(() => {
+      const cardsArray = this.state.cards
+      console.log(cardsArray.length);
```

Comment:
I like that you've kept all the API interaction logic in one place - the callbacks are a little more complex, but I would say it makes the app as a whole much easier to comprehend. Whether or not you intended it, this is a great example of the container component pattern well-applied.